app/models/company.rb

```diff
@@ -0,0 +1,12 @@
+class Company < ApplicationRecord
+    VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/.freeze
+    VALID_PHONE = /\A\d{10}\z/.freeze
+    enum COMPANY_TYPE: { water_supply: 0, heating: 1, rent_payment: 2, accommodation_payment: 3,
```

Comment:
where are constants for each type?

Reply:
WATER_SUPPLY = 1
  HEATING = 2
 RENT_PAYMENT = 3
ACCOMMODATION_PAYMENT = 4
GAS = 5
ELECTRICITY = 6
GARBAGE_REMOVAL = 7
INTERNET = 8
INTERCOM = 9
TV = 10
OTHER = 11
enum COMPANY_TYPE: { WATER_SUPPLY => 'water_supply', HEATING => 'heating', RENT_PAYMENT => 'rent_payment' ....... }

Reply:
```
  INSURER = "insurer"
  ADJASTOR = "adjastor"

  ROLES = [
    INSURER,
    ADJASTOR
  ]

  enum role: ROLES

```

Reply:
an example